typescript/examples/eleven-labs/conversational-agent/README.md

```diff
@@ -2,7 +2,7 @@
 
 This example uses the ElevenLabs base project described [here](https://elevenlabs.io/docs/conversational-ai/guides/conversational-ai-guide-nextjs) and follows the steps to add client tool calling described [here](https://elevenlabs.io/docs/conversational-ai/customization/client-tools).
 
-https://github.com/user-attachments/assets/67c7f2d2-0b6a-4e0a-b24b-423e35a94c69
+https://github.com/user-attachments/assets/2a53b2af-eefe-46c7-a3d4-9441eefe0f81
 
```

Comment:
Id explain what we're building. This is describing more how we build it, which is harder to parse

Sometihng on the lines of

This example builds a conversational agent with ElevenLabs, that can query the balance in your wallet and recommend you which tokens to buy, with some special touch ✨ 

It is built on top of the ElevenLabs base project, and implements both functionalities using tool calling.